src/mmw/js/src/draw/views.js

```diff
@@ -213,9 +213,10 @@ var PlaceMarkerView = Marionette.ItemView.extend({
         'change:toolsEnabled': 'render'
     },
 
-    onItemClicked: function() {
+    onItemClicked: function(e) {
         var self = this,
             map = App.getLeafletMap(),
+            itemName = e.target.innerHTML,
```

Comment:
This is a minor thing, but is it true that the stuff that comes out of `e.target.innerHTML` is html whereas the location names (in this case `itemName`) are assumed to be plain text?  If so, it possible that at some point something changes so that '<b>Big City</b>' appears as the location?  (Once again, very minor.)